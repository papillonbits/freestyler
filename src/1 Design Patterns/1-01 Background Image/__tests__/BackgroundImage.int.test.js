import { regular } from './BackgroundImage.int.story'

describe('<BackgroundImage />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
