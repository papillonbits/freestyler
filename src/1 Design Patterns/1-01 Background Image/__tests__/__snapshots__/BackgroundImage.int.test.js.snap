// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackgroundImage /> Render must match regular() 1`] = `
<div>
  <div
    class="background-image"
  >
    <h1>
      Background Image
    </h1>
    <div
      class="background-image__image"
    />
  </div>
</div>
`;
